css/builder.css

```diff
@@ -551,6 +551,18 @@ input[type="time"]:not(.basic-time-picker)::-webkit-calendar-picker-indicator {
   opacity: 0;
 }
 
+.fl-date-range {
```

Comment:
Could these have been in the JS API too? The question is: can the JS API UI work/render as expected without these CSS?

Reply:
Yes, JS API can render the UI